Zappy_IA_Report/DTC/Video/Tutorial2.py

```
@@ -0,0 +1,29 @@
+
+from sklearn.preprocessing import StandardScaler
+
+scaleStandard = StandardScaler()
+X1 = scaleStandard.fit_transform(X1)
+X1 = pd.DataFrame(X1, columns=['YRS' ,'G' ,'AB' ,'R' ,'H' ,'2B' ,'3B' ,'HR' ,'RBI' ,'BB','SO' ,'SB' ,'BA', 'HOF'])
+print("\nX1.head: ", X1.head())
+print("\nX1.describe: ", X1.describe().round(3))
+
+from sklearn.preprocessing import MinMaxScaler
+
+scaleMinMax = MinMaxScaler(feature_range=(0,1))
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)